pkg/ovncontroller/daemonset.go

```diff
@@ -27,30 +26,19 @@ import (
 	"k8s.io/utils/ptr"
 )
 
-// DaemonSet func
-func DaemonSet(
+func CreateOVNDaemonSet(
```

Comment:
these should be renamed because they don't create anything, just prepare an object.